remme/ws/constants.py

```diff
@@ -15,6 +15,21 @@
 
 from enum import Enum, IntEnum, unique
 
+# to be replaced with a family name if applicable
+ATOMIC_SWAP = 'atomic-swap'
```

Comment:
I do not see a reason for this abstraction

Reply:
One needs to identify a family of events